termpair/server.py

```diff
@@ -159,31 +158,30 @@ def _gen_terminal_id(ws: WebSocket) -> TerminalId:
 @app.websocket_route("/connect_to_terminal")
 async def connect_to_terminal(ws):
     try:
-        async with ws:
-            terminal_id = _gen_terminal_id(ws)
-            data = await ws.receive_json()
-            terminal = Terminal(
-                ws=ws,
-                web_clients=set(),
-                rows=data["rows"],
-                cols=data["cols"],
-                allow_browser_control=data["allow_browser_control"],
-                command=data["command"],
-                broadcast_start_time_iso=data["broadcast_start_time_iso"],
-            )
-            terminals[terminal_id] = terminal
+        terminal_id = _gen_terminal_id(ws)
+        data = await ws.receive_json()
+        terminal = Terminal(
+            ws=ws,
+            web_clients=set(),
+            rows=data["rows"],
+            cols=data["cols"],
+            allow_browser_control=data["allow_browser_control"],
+            command=data["command"],
+            broadcast_start_time_iso=data["broadcast_start_time_iso"],
+        )
+        terminals[terminal_id] = terminal
 
-            await ws.send(
-                json.dumps({"event": "start_broadcast", "payload": terminal_id})
-            )
+        await ws.send(
+            json.dumps({"event": "start_broadcast", "payload": terminal_id})
+        )
 
-            t1 = asyncio.ensure_future(_forward_terminal_data_to_web_clients(terminal))
-            done, pending = await (
-                asyncio.wait([t1], return_when=asyncio.FIRST_COMPLETED)
-            )
-            for task in pending:
-                task.cancel()
-    except Exception:
```

Comment:
Why was this taken out?

Reply:
It seems to me your were using it to show the exception in the console in case something unexpected goes wrong — and indeed pre-0.13.1 Bocadillo didn’t do so by default which I personally felt to be a discrepancy too. In 0.13.1 Bocadillo shows the error traceback, so I deemed these global error catches were not necessary. Did they serve another purpose?

Reply:
Ah, no. If Bocadillo shows the traceback, then even better. Sounds like Bocadillo has all the right defaults and abstractions.